mlir/include/mlir/Dialect/LLVMIR/LLVMIntrinsicOps.td

```diff
@@ -842,22 +842,26 @@ def LLVM_GetActiveLaneMaskOp
 /// Create a call to Masked Load intrinsic.
 def LLVM_MaskedLoadOp : LLVM_OneResultIntrOp<"masked.load"> {
   let arguments = (ins LLVM_AnyPointer:$data, LLVM_VectorOf<I1>:$mask,
-                   Variadic<LLVM_AnyVector>:$pass_thru, I32Attr:$alignment);
+                   Variadic<LLVM_AnyVector>:$pass_thru, I32Attr:$alignment,
+                   DefaultValuedAttr<BoolAttr, "false">:$nontemporal);
```

Comment:
```suggestion
                   UnitAttr:$nontemporal);
```
I think in other places in LLVM dialect we use UnitAttr. Would this work here as well?

Reply:
Hi @gysit , thanks for the review! I tried your code, but I got:
```
 no known conversion from 'bool' to '::mlir::UnitAttr'
```
I noticed that in `LLVMIntrinsicOps.td` there is no use of `UnitAttr` . I think because we use the `$_int_attr($alignment)` we need to have a `UnitAttr` passed (instead of a bool).

Reply:
Ah and we use `_int_attr` because we need to match the alignment

Reply:
Also, with the `UnitAttr`, when I don't pass the attr, I get ` candidate function not viable: requires 8 arguments, but 7 were provided`

Reply:
That worked! I updated the code, thanks @gysit !